src/app/containers/CpsRelatedContent/RelatedContentPromoList/index.jsx

```diff
@@ -35,7 +33,7 @@ const RelatedContentPromoList = ({ promoItems, dir, isMediaContent }) => {
         group4: 8,
         group5: 8,
       }}
-      as={isMediaContent ? MostWatchedOl : StoryPromoUl}
+      as={isMediaContent ? <MostWatchedOl role="list" /> : StoryPromoUl}
```

Comment:
Is this implementation OK or should it be receiving children, rather than opening & closing the component in one declaration?

Reply:
Yeh something doesn't seem right here. Worth looking into

Reply:
Aye fixed this as part of the audit/bug-shooting PR https://github.com/bbc/simorgh/pull/8174/files#diff-9648cbb126e6fd0f5cf1ce1339a624d4498186ab8ab30813d19bf955f1d595bcR40